.github/workflows/check.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Install tools
       run: |
         pip install -U pip
-        pip install tox tox-gh-actions coveralls
+        pip install tox tox-gh-actions coveralls coverage[toml]
```

Comment:
This is derived from #446; I can't get the `check.yml` action to pass without it.